crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/rpc.rs

```diff
@@ -424,6 +424,24 @@ impl LedgerRpcProvider for LedgerDB {
             None => Ok(None),
         }
     }
+
+    fn get_head_soft_batch(&self) -> Result<Option<SoftBatchResponse>, anyhow::Error> {
+        let next_ids = self.get_next_items_numbers();
+        let next_soft_batch = next_ids.soft_batch_number;
```

Comment:
Code style: I would expect the beginning of this method to start as it did in `get_head_soft_batch_height` - where you subtract 1 from soft_batch_number, not inside the call to .db.get